cli/certs.go

```diff
@@ -1,54 +1,57 @@
 package cli
 
 import (
-	"errors"
 	"strconv"
 
 	"github.com/spf13/cobra"
 )
 
-var cmdCerts = []cobra.Command{
-	cobra.Command{
+// NewCertsCmd returns certificate command.
+func NewCertsCmd() *cobra.Command {
+	var keySize uint16
+	var keyType string
+	var ttl uint32
+
+	issueCmd := cobra.Command{
 		Use:   "issue",
-		Short: "issue <thing_id> <keybits> <keytype> <hoursvalid> <user_auth_token>",
+		Short: "issue <thing_id> [--keysize=2048] [--keytype=rsa] [--ttl=<hoursvalid>]",
```

Comment:
Why are `keysize` and `keytype` values hardcoded in this example, and `ttl` is not? For the consistency sake, either represent all with their default values, or none.